scripts/rust-vmm-consistency-check.py

```diff
@@ -0,0 +1,75 @@
+import subprocess
+import json
+from collections import defaultdict
+
+def get_cargo_metadata():
+    result = subprocess.run(
+        ['cargo', 'metadata', '--format-version=1'],
+        capture_output=True,
+        text=True
+    )
+
+    if result.returncode != 0:
+        exit(1)
+
+    metadata = json.loads(result.stdout)
+    return metadata
+
+def find_dependents_of_rust_vmm(metadata):
+    rust_vmm_packages = defaultdict(list)
+    direct_dependents = defaultdict(list)
+
+    # Identify packages from `rust-vmm` and record version
+    for pkg in metadata['packages']:
+        repository = pkg['repository'] or ''
+        if 'github.com/rust-vmm' in repository:
```

Comment:
Can we have this variable(github.com/rust-vmm() as the input  to the program so that in future we can reuse the script for other usage. If you agree with this then we could just rename the script as well.

Reply:
Sure, it's always a good idea to generalize, I will change this the first thing tomorrow morning :)